core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -212,11 +213,28 @@ public void getReport() {
   }
 
   @Test
-  public void getStoreMeta() {
+  public void getStoreMeta() throws Exception {
     mBlockWorker.getStoreMeta();
-    mBlockWorker.getStoreMetaFull();
     verify(mBlockStore, times(2)).getBlockStoreMeta(); // 1 is called at metrics registration
-    verify(mBlockStore).getBlockStoreMetaFull();
+    long blockId1 = mRandom.nextLong();
+    long blockId2 = mRandom.nextLong();
+    long sessionId = mRandom.nextLong();
+    mBlockWorker.createBlock(sessionId, blockId1, 0, "", 1L);
+    // Available space in `MEM` is not enough. Block will be allocated in `HDD`.
+    mBlockWorker.createBlock(sessionId, blockId2, 0, "", 1024 * 1024 * 1024L);
+    BlockStoreMeta storeMeta = mBlockWorker.getStoreMetaFull();
+    assertEquals(2, storeMeta.getBlockList().size());
+    assertEquals(2, storeMeta.getBlockListByStorageLocation().size());
+    assertEquals(0, storeMeta.getBlockList().get("MEM").size());
+    // 1GB (`MEM`) + 2GB (`HDD`)
+    assertEquals(3221225472L, storeMeta.getCapacityBytes());
+    // 512MB * 2 (reserved space) + 1bit (`MEM`) + 1GB (`HDD`)
+    assertEquals(2147483649L, storeMeta.getUsedBytes());
+    mBlockWorker.commitBlock(sessionId, blockId1, true);
+    mBlockWorker.commitBlock(sessionId, blockId2, true);
+    storeMeta = mBlockWorker.getStoreMetaFull();
+    assertEquals(1, storeMeta.getBlockList().get("MEM").size());
```

Comment:
we can't assert usedBytes after commit block cause the real path size is 0 in UT.
see BlockMetadataManager#commitTempBlockMeta() -> dir.addBlockMeta(block) // block size is 0 here